testgrid/cmd/gatherer/testdata/Comments.json

```diff
@@ -0,0 +1,250 @@
+      "received_events_url": "https://api.github.com/users/stevekuznetsov/received_events",
+      "type": "User",
+      "site_admin": false
+    },
+    "created_at": "2019-05-22T16:56:25Z",
+    "updated_at": "2019-05-22T16:56:25Z",
+    "author_association": "CONTRIBUTOR",
+    "body": "Yeah so that confirms it, the status existed, was failed and was not handled by `trigger` otherwise, so `skip` ignored it because it was a required status (required-if-present)"
+  },
+  {
+    "url": "https://api.github.com/repos/kubernetes/test-infra/issues/comments/494892456",
+    "html_url": "https://github.com/kubernetes/test-infra/issues/12740#issuecomment-494892456",
+    "issue_url": "https://api.github.com/repos/kubernetes/test-infra/issues/12740",
+    "id": 494892456,
+    "node_id": "MDEyOklzc3VlQ29tbWVudDQ5NDg5MjQ1Ng==",
+    "user": {
+      "login": "liggitt",
+      "id": 980082,
+      "node_id": "MDQ6VXNlcjk4MDA4Mg==",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/980082?v=4",
+      "gravatar_id": "",
+      "url": "https://api.github.com/users/liggitt",
+      "html_url": "https://github.com/liggitt",
+      "followers_url": "https://api.github.com/users/liggitt/followers",
+      "following_url": "https://api.github.com/users/liggitt/following{/other_user}",
+      "gists_url": "https://api.github.com/users/liggitt/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/liggitt/starred{/owner}{/repo}",
+      "subscriptions_url": "https://api.github.com/users/liggitt/subscriptions",
+      "organizations_url": "https://api.github.com/users/liggitt/orgs",
+      "repos_url": "https://api.github.com/users/liggitt/repos",
+      "events_url": "https://api.github.com/users/liggitt/events{/privacy}",
+      "received_events_url": "https://api.github.com/users/liggitt/received_events",
+      "type": "User",
+      "site_admin": false
+    },
+    "created_at": "2019-05-22T17:16:43Z",
+    "updated_at": "2019-05-22T17:16:43Z",
+    "author_association": "MEMBER",
+    "body": "> because it was a required status (required-if-present)\r\n\r\nhow is that determined? is that an attribute of the job? I didn't see anything in the job spec that looked like it was required"
+  },
+  {
+    "url": "https://api.github.com/repos/kubernetes/test-infra/issues/comments/494988860",
+    "html_url": "https://github.com/kubernetes/test-infra/issues/12740#issuecomment-494988860",
+    "issue_url": "https://api.github.com/repos/kubernetes/test-infra/issues/12740",
+    "id": 494988860,
+    "node_id": "MDEyOklzc3VlQ29tbWVudDQ5NDk4ODg2MA==",
+    "user": {
+      "login": "stevekuznetsov",
+      "id": 7328370,
+      "node_id": "MDQ6VXNlcjczMjgzNzA=",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/7328370?v=4",
+      "gravatar_id": "",
+      "url": "https://api.github.com/users/stevekuznetsov",
+      "html_url": "https://github.com/stevekuznetsov",
```

Comment:
ruh roh

Reply:
Test cases that aren't "real historical data, but slightly modified" are in the works.